apps/remix-ide/src/remixAppManager.js

```diff
@@ -70,13 +70,6 @@ export class RemixAppManager extends PluginManager {
     this.event.emit('deactivate', plugin)
   }
 
-  onRegistration () {}
-
-  async ensureActivated (apiName) {
-    await this.activatePlugin(apiName)
-    this.event.emit('ensureActivated', apiName)
-  }
-
   async ensureDeactivated (apiName) {
```

Comment:
and what about this?

Reply:
Will work on that in another PR. I didn't wanted to be too eager ;)